packages/headless/doc-parser/doc-parser.ts

```diff
@@ -265,8 +266,11 @@ const controllers: ControllerConfiguration[] = [
   },
 ];
 
-const result = controllers.map((controller) =>
-  resolveController(entryPoint, controller)
-);
+const result = {
+  controllers: controllers.map((controller) =>
+    resolveController(entryPoint, controller)
+  ),
+  actions: resolveActionNamespaces(entryPoint),
```

Comment:
As mentioned in the description, every action is parsed in this PR. After this PR, I'll most likely add a configuration section similar to the controllers' so we can attach samples.

On the subject of samples, feel free to comment below where you think we should write samples:
* In comments?
* As sibling files?
* In the React samples?

Reply:
Are samples needed for actions?

Reply:
Not necessarily, but I believe some actions are meant to be used with helpers? For those specifically it may be nice.